atlasexec/atlas.go

```diff
@@ -63,13 +63,20 @@ type (
 		RevisionsSchema string
 		Vars            Vars
 	}
+	LintContextInput struct {
```

Comment:
maybe we should go with just `LintContext` because there is not output, 
more important to stay consistent though, if we use the same naming somewhere else so lets keep that way